docs/economics/solution-and-estimation.rst

```diff
@@ -39,9 +40,13 @@ partly uncertain due to the unknown realization of the shocks:
 
 .. math::
 
-     E\max(S(t)) =\hspace{11cm}
-
-    \int_{\epsilon_1(t)} ... \int_{\epsilon_K(t)}\max\{R_1(t), ..., R_K(t)\}f_{\epsilon}(\epsilon_1(t), ... ,\epsilon_K(t))d\epsilon_1(t) ... d\epsilon_K(t),
+    E\max(S(t)) &= \int_{\epsilon_1(t)} \dots \int_{\epsilon_K(t)} \max\{ \\
+                &R_1(t) + \delta E[V(S(t+1) | S(t), d_1(t) = 1], \\
```

Comment:
- The number of opening braces does not match the number of closing braces for V(S(t+1) 
 - V(S(t+1) needs to be V(S(t + 1), t + 1), right?
- Why not use EMAX in the precise notation defined above directly?